RFS/src/test/java/org/opensearch/migrations/parsing/BulkResponseParserTest.java

```diff
@@ -39,7 +39,7 @@ void testFindSuccessDocs_truncatedResponse() throws IOException {
         var successDocId = "223";
```

Comment:
Looks like I broke this test case when I changed the generated output slightly, need to fix this...